### ISSUE TITLE
Watch mode looking for --template in cli while config passed with all required setup.

### ISSUE THREAD

**Issue by RIP21** [dotansimha/graphql-code-generator#909]
```
gql-gen --config ./codegen-client.yml --watch
```
Results in 
```
Please specify language/platform, using --template flag!
```
Using without `--watch` works just fine.

**Comment by dotansimha** [comment]
I was fixes recently in  https://github.com/dotansimha/graphql-code-generator/pull/906, use the latest alpha :)

**Comment by dotansimha** [comment]
Fixed in 0.14.2 🎉
